t/v2/dir.t

```diff
@@ -210,6 +210,7 @@ checked val as expect: b
             check_res(res, err, nil, nil, true)
 
             local cur_time = ngx.now()
+            ngx.update_time()
```

Comment:
call `ngx.update_time()` before `ngx.now()`

Reply:
`ngx.update_time()` must call between `ngx.now()`, because of the code `ngx.say("err: ", err, ", more than 1sec: ", ngx.now() - cur_time > 1)` at line `215`

Reply:
got it